rotkehlchen/tests/unit/test_aave.py

```diff
@@ -40,3 +59,64 @@ def test_atoken_to_asset():
 
     for atokenv1, reserve_asset in ATOKENV1_TO_ASSET.items():
         assert atoken_to_asset(atokenv1.resolve_to_evm_token()) == reserve_asset
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12']])
+@pytest.mark.parametrize('should_mock_price_queries', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+@pytest.mark.parametrize('default_mock_price_value', [ONE])
+def test_aave_v1_events_stats(
+        database: 'DBHandler',
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_transaction_decoder: 'EthereumTransactionDecoder',
+        rotki_premium_credentials: 'PremiumCredentials',
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
+        price_historian: 'PriceHistorian',  # pylint: disable=unused-argument
+):
+    tx_hashes = [
+        deserialize_evm_tx_hash('0x8b72307967c4f7a486c1cb1b6ebca5e549de06e02930ece0399e2096f1a132c5'),  # supply    102.92 DAI and get same amount of aDAI  # noqa: E501
+        deserialize_evm_tx_hash('0x78ae48d93e0284d1f9a5e1cd4a7e5f2e3daf65ab5dafb0c4bd626aa90e783d60'),  # supply    160    DAI  # noqa: E501
```

Comment:
this test uses the same data as one we had before. The events are all lefteris events on v1